src/content/contributing/translation-program/index.md

```diff
@@ -1,181 +1,99 @@
+### Our progress so far {#our-progress}
 
-**Our vision**
-
-- Translate essential content for members of the Ethereum community from as many countries and parts of the world as possible
-- Support knowledge sharing across languages to create a better informed and educated Ethereum community
-- Increase Ethereum's inclusivity and accessibility by removing the language barriers preventing non-English speakers from joining the ecosystem
+- [**3,700 +** translators](/contributing/translation-program/contributors/)
+- [**46** languages live on site](/languages/)
+- [**2.9 million** words translated in 2021](/contributing/translation-program/acknowledgements/)
+- [**1.9 million** words translated in 2022 so far](/contributing/translation-program/acknowledgements/)
 
-## How to translate {#how-to-translate}
+If you want to get involved and help us grow the global Ethereum community by translating the website into your language, follow the steps below!
 
-<InfoBanner shouldCenter emoji=":light_bulb:">
-  We encourage you to check out the ethereum.org Translation Style Guide. It contains some of the most important guidelines, instructions, and tips for translators to help localize the website.
-  {" "}<Link to="/contributing/translation-program/translators-guide/">View the Translation Style Guide</Link>
+<InfoBanner emoji=":tada:">
+  Check out our <Link to="/contributing/translation-program/acknowledgements/">Translator Acknowledgements</Link> page, and{" "}
+  claim your POAP token! If you translated ethereum.org in 2022, there's a unique POAP waiting for you.{" "}
+  <Link to="/contributing/translation-program/acknowledgements/#poap">More on POAPs</Link>
 </InfoBanner>
 
-### Visual guide {#visual-guide}
-
-For more visual learners, watch Luka walk through getting set up with Crowdin. Alternatively, you can find the same steps in a written format in the next section.
-
-<YouTube id="Ii7bYhanLs4" />
-
-### Written guide {#written-guide}
-
-1. **[Join our project on Crowdin](https://crowdin.com/project/ethereum-org/invite)**  
-   You will need to log in to your Crowdin account or sign up if you don’t already have one. All that is required to sign up is an e-mail account and password.
+### What we're working on {#our-projects}
 
-2. **Open the language you want to translate and find a document to work on**  
-   The website content is divided into a number of documents and content buckets. You can check the progress of each document on the right – if translation progress is below 100%, please contribute!
+Our community of contributors is actively translating multiple high-impact websites and resources for the global Ethereum community.
 
-   Don't see your language listed? [Open an issue](https://github.com/ethereum/ethereum-org-website/issues/new/choose) or ask in our [Discord](https://discord.gg/6WX7E97)
+#### Ethereum.org {#ethereumorg}
```

Comment:
FYI I added this "ethereum.org" section - I think it's worth re-iterating this is one of the projects we work on.

I also added a description of each project, which I encourage edits on :)